components/docs-chef-io/content/automate/centralizing_log.md

```diff
@@ -144,3 +144,16 @@ Restart your splunk forwarder using the following command:
 ```
 
 The above step will let you view the `automate.log` file in your splunk forwarder. Click [here](https://statics.teams.cdn.office.net/evergreen-assets/safelinks/1/atp-safelinks.html) to know about the Splunk Forwarder.
+
+## Frequently Asked Questions
+
+### Is the setting of `redirect_log_file_path` directory where the log files will be stored or the full path of the actual log file?
+
+- `redirect_log_file_path` is a directory where the automate.log file will be created.
+
+### Patching the log config. Should we run one command to patch the whole cluster from the bastion host?
+
+- Run multiple commands, e.g.:
```

Comment:
We should include all the steps with more details. 
- Mount an NFS or network-attached storage to all the nodes. Choose a log location
- command to patch Automate nodes with the same log location
- command to patch Server nodes with the same log location
- command to patch PG nodes with the same log location
- command to patch OS nodes with the same log location